system/HTTP/Files/FileCollection.php

```diff
@@ -111,6 +111,41 @@ public function getFile(string $name)
 
 	//--------------------------------------------------------------------
 
+	/**
+	 * Verify if a file exist in the collection of uploaded files and is have been uploaded with multiple option.
+	 *
+	 * @param string $name
+	 *
+	 * @return array|null
+	 */
+	public function getFileMultiple(string $name)
+	{
+		$this->populateFiles();
+
+		if ($this->hasFile($name))
+		{
+			if (strpos($name, '.') !== false)
```

Comment:
Pretty sure this needs an `is_string()` check before. An array would throw an exception under `strpos`.

Reply:
It doesnt need is_string() because $name is a string not an arrary.

Reply:
Doh! You're correct. Sorry, confused it when I saw the `is_array` check below. My bad for scanning too quickly.